usaspending_api/awards/v2/views/award_spending.py

```diff
@@ -15,7 +15,6 @@
 class AwardTypeAwardSpendingViewSet(CachedDetailViewSet):
     """
     Return all award spending by award type for a given fiscal year and agency id
-    endpoint_doc: award_spending/award_category.md
```

Comment:
I'm confused why this endpoint doesn't need a doc

Reply:
It's not being used anywhere.